source/patterns/00-atoms/block/image/image.twig

```diff
@@ -6,6 +6,8 @@
     <source {{- html.attributes(source) }}>
   {%- endfor -%}
 
+  {%- set image = image|default({})|merge({class: image.class|default([]) }) -%}
```

Comment:
Doesn't do anything?

Reply:
It's not needed, it will work if not there.

Reply:
Okay, makes sense to delete this line then 👍